packages/cli/src/commands/serve.js

```diff
@@ -46,10 +52,14 @@ export const builder = async (yargs) => {
       builder: (yargs) =>
         yargs.options({
           port: {
-            default: getConfig().web?.port || 8910,
+            default: redwoodProjectConfig.web?.port || 8910,
             type: 'number',
             alias: 'p',
           },
+          host: {
+            default: redwoodProjectConfig.web?.host || 'localhost',
```

Comment:
Question here is: if the user configures different hosts for api and web, do they expect `yarn rw serve` to start both on different hosts at the same time? Or one host like we have here?

Reply:
Sorry I didn't mean to imply that you had made this change, I just wanted to ask the question. And you're right about the other options too. And I don't want to introduce too much scope since this PR was originally just supposed to fix a bug. Agree that 1) is less disruptive; my priority at the moment is not making this behaviorally breaking